src/core/config/descriptor_providers/xlio_config_schema.json

```diff
@@ -0,0 +1,1454 @@
+                                    "description": "Maximum data size sent inline. Setting to 0 disables inlining. Maps to XLIO_TX_MAX_INLINE environment variable. Max send inline data set for QP. Data copied into the INLINE space is at least 32 bytes of headers and the rest can be user datagram payload. XLIO_TX_MAX_INLINE=0 disables INLINEing on the Tx transmit path. In older releases this parameter was called: XLIO_MAX_INLINE."
+                                },
+                                "rx_global_array_size": {
+                                    "type": "integer",
+                                    "default": 16000,
+                                    "minimum": 0,
+                                    "title": "RX WRE global array size",
+                                    "description": "Number of Work Request Elements allocated in all RQs. Maps to XLIO_RX_WRE environment variable. "
+                                },
+                                "rx_batch_size": {
+                                    "type": "integer",
+                                    "default": 1024,
+                                    "minimum": 1,
+                                    "maximum": 1024,
+                                    "title": "RX WRE batch size",
+                                    "description": "Number of Work Request Elements and RX buffers to batch before recycling. Maps to XLIO_RX_WRE_BATCHING environment variable. Batching decrease latency mean, but might increase latency STD.\nValue range is 1-1024."
+                                }
+                            }
+                        },
+                        "tcp_segments": {
+                            "type": "object",
+                            "description": "TCP segmentation settings.",
+                            "properties": {
+                                "socket_batch_size": {
+                                    "type": "integer",
+                                    "default": 64,
+                                    "minimum": 1,
+                                    "title": "Socket segment batch size",
+                                    "description": "Number of TCP segments fetched from segments pool by a socket at once. Maps to XLIO_TX_SEGS_BATCH_TCP environment variable. Min value is 1"
+                                },
+                                "ring_batch_size": {
+                                    "type": "integer",
+                                    "default": 1024,
+                                    "minimum": 1,
+                                    "title": "Ring segment batch size",
+                                    "description": "Number of TCP segments fetched from segments pool by a ring at once. Maps to XLIO_TX_SEGS_RING_BATCH_TCP environment variable. Min value is 1"
+                                },
+                                "pool_batch_size": {
```

Comment:
This parameter is unused and should b removed

Reply:
True - but Out of scope for this PR.
We can open an issue to solve it (didn't want to remove members in sys_vars.cpp in this PR to affect data-path)

Reply:
do you think we should simply not set it? I wanted to keep the PR scope to converting legacy to new config - but maybe we can add a bit of extra logic